### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by NonTerraePlusUltra** [hli1221/imagefusion-rfn-nest#6]
Hello Dr.Li:

It's very happy to study your work，but I have doubts about the details of the loss function.
 "Input and Output denote the input image and the output image (both indicate one image)"What does it mean that the input and output are an image when training the Auto-encoder Network?
I don't know how to understand this part, please help me, thank you!.

Best Regards.

**Comment by hli1221** [comment]
Hi, thanks for your attention. The 'one image' means one modality data. For training phase, it means visible image.

**Comment by NonTerraePlusUltra** [comment]
> Hi, thanks for your attention. The 'one image' means one modality data. For training phase, it means visible image.

Thank you for your answer. I have another question. Is the output image reconstructed from the input image after extracting features?

**Comment by hli1221** [comment]
For the training phase, YES. That is why it's called autoencoder.

**Comment by NonTerraePlusUltra** [comment]
> 对于培训阶段，是的。这就是为什么它被称为自动编码器。

thanks for your answer！
